juriscraper/opinions/united_states/state/__init__.py

```diff
@@ -167,6 +167,8 @@
     "va",
     "vactapp_u",
     "vactapp_p",
+    "visuper_p",
+    "visuper_u",
```

Comment:
This is great - but its in the wrong place - these files should be in the territories section